src/status_im2/contexts/quo_preview/share/qr_code.cljs

```diff
@@ -1,55 +1,105 @@
 (ns status-im2.contexts.quo-preview.share.qr-code
-  (:require [react-native.core :as rn]
+  (:require [quo.components.text :as text]
```

Comment:
please update components to use `quo.core` here if you're making adjustments 👍

Reply:
Done! good catch!